molgenis-api-tests/src/test/java/org/molgenis/api/tests/utils/RestTestUtils.java

```diff
@@ -131,6 +131,21 @@ public static String uploadEMX(String adminToken, String fileName) {
     return uploadEMXFile(adminToken, file);
   }
 
+  /**
+   * Import emx file using add/update.
+   *
+   * <p>Importing is done async in the backend, but this methods waits for importing to be done.
+   *
+   * @param adminToken to use for login
+   * @param fileName the file to upload
+   * @return String indicating state of completed job
+   */
+  public static String uploadEMXFileWithoutPackage(String adminToken, String fileName) {
```

Comment:
Convention: uploadEMXFileWithoutPackage -> uploadEmxFileWithoutPackage

Reply:
There is a lot of this going around in this class, but the renaming should not be part of this PR I think... changed this method anyhow...